src/examples/scatterplot.jsx

```diff
@@ -56,15 +56,15 @@ class ScatterPlotExample extends Component {
   renderCustomPoints() {
     return (
       <div>
-        <h1>Custom Points</h1>
+        <h2>Custom Points</h2>
         <JubilationChart height={300} width={600}>
           <JubilationScatter
             data={this.state.points}
             color={'red'}
             size={10}
             style={{ opacity: 0.5 }}
           />
-          <XAxis numTicks={2} title="X axis" />
+          <XAxis numTicks={4} title="X axis" extendTicks={true} />
```

Comment:
thats fine with me. As an FYI, you can get away with just the attribute name in JSX for booleans that you want to set to true.
```jsx
<XAxis numTicks={4} title="X axis" extendTicks />
```